packages/base/Select/package.json

```
@@ -43,10 +43,14 @@
   ],
   "peerDependencies": {
     "@material-ui/core": "4.12.4",
+    "@toptal/picasso-tailwind": "2.6.0",
```

```suggestion
    "@toptal/picasso-tailwind": ">=2.6.0",
```

as we decided in https://toptal-core.atlassian.net/wiki/spaces/FE/pages/3835068492/PI-3379+Decision+Log